redash/query_runner/google_spreadsheets.py

```diff
@@ -100,18 +100,20 @@ def __init__(self, worksheet_title):
 def parse_query(query):
     values = query.split("|")
     key = values[0]  # key of the spreadsheet
-    worksheet_num_or_title = 0  # A default value for when a number of inputs is invalid
+    worksheet_pointer = ' 1'  # A default value for when a number of inputs is invalid
```

Comment:
@immortalcodes hey, thanks for updating this PR. I think I might have misspoken though, or I wasn't clear in my communication - we want to separate `parse_query` into two versions of a similar function, where one uses a worksheet number and the other uses the title, and we want to call the correct function depending on what the user inputs.

Reply:
Ideally, we don't have to worry about things like a space prefix on a string signifying anything; we should keep the data as separate as possible from the code logic.